header.php

```diff
@@ -70,45 +72,27 @@
   <header id="header">
     <div class="container padding-top-tiny">
       <div class="row">
-        <div class="col col-s-12 col-m-3">
-          <h1 class="font-uppercase"><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
+        <div class="col col-s-9 col-l-3">
+          <h1 class="font-uppercase"><a href="<?php echo site_url('/home'); ?>"><?php bloginfo('name'); ?></a></h1>
         </div>
-        <div class="col col-s-12 col-m-5">
-          <ul id="header-menu" class="font-sans u-inline-list">
-            <?php
-              if (!is_page('home')) {
-            ?>
-            <li><a href="<?php echo site_url('/sobre-nosotros'); ?>">Home</a></li>
-            <?php
-              }
-              global $post;
-              if ($post && is_single_type('programacion', $post)) {
-            ?>
-            <?php
-                $programacion_posts = get_posts('post_type=programacion&posts_per_page=3');
-                if ($programacion_posts) {
-                  foreach ($programacion_posts as $programacion) {
-                    $number = get_post_meta($programacion->ID, '_igv_number', true);
-            ?>
-            <li><a href="<?php echo the_permalink($programacion->ID); ?>" <?php echo $programacion->ID == $post->ID ? 'class="font-italic"' : ''; ?>>No. <?php echo $number;?></a></li>
-            <?php
-                  }
-                }
-            ?>
-            <li><a href="<?php echo site_url('/programacion'); ?>">+</a></li>
-            <?php
-              } else if (!is_page('home') && !is_post_type_archive('programacion')) {
-            ?>
-            <li><a href="<?php echo get_the_permalink(get_id_by_slug('sobre-nosotros')); ?>#prensa">Prensa</a></li>
-            <li><a href="<?php echo get_the_permalink(get_id_by_slug('sobre-nosotros')); ?>#contacto">Contacto</a></li>
-            <li><a href="<?php echo site_url('/noticias'); ?>">Noticias</a></li>
-            <?php
-              }
-            ?>
+        <div class="col col-s-3 only-mobile text-align-right">
+          <div class="menu-toggle u-pointer u-inline-block"><span class="menu-open"><?php 
+            echo file_get_contents(get_bloginfo('stylesheet_directory') . '/img/dist/ui-menu.svg'); 
+          ?></span><span class="menu-close"><?php 
+            echo file_get_contents(get_bloginfo('stylesheet_directory') . '/img/dist/ui-close.svg'); 
+          ?></span></div>
+        </div>
+        <div class="col col-s-12 col-l-5">
+          <ul id="header-menu" class="font-sans">
+            <li><a href="<?php echo site_url('/nosotros'); ?>"><?php _e('[:es]Nosotros[:en]About'); ?></a></li>
```

Comment:
the page on the existing site is still with the slug sobre-nosotros. thats a better url no?